docs/getting-started.rst

```diff
@@ -165,6 +165,19 @@ to set :makevar:`CPPFLAGS` in :file:`make/local`:
 
     CPPFLAGS+=-DSTAN_MATH_CONSTRAINT_TOLERANCE=1e-5
 
+
+Using External C++ Code
+_______________________
+
+BridgeStan supports the same `capability to plug in external C++ code as CmdStan <https://mc-stan.org/docs/cmdstan-guide/external_code.html>`_.
+
+Namely, you can declare a function in your Stan model and then define it in a separate C++ file.
+This requires passing the ``--allow-undefined`` flag to the Stan compiler when building your model.
+The :makevar:`USER_HEADER` variable must point to the C++ file containing the function definition.
```

Comment:
Should we say explicitly that it should be a C++ header file?  We only reference it being a `.hpp` file in the makevar variable name and the example file name.

Reply:
It technically does not need to be a header (or even a `.hpp`, in theory a `.txt` would also work, since the compiler is just pasting it in), as long as it's code that can be included into the model's translation unit it would be fine. 

Personally I think the name `USER_HEADER` is evocative enough